server/config.go

```diff
@@ -398,6 +398,7 @@ var defaultSchedulers = SchedulerConfigs{
 	{Type: "balance-region"},
 	{Type: "balance-leader"},
 	{Type: "hot-region"},
+	{Type: "fail-over"},
```

Comment:
after upgrading from existed pd, the scheduler won't be added automatically. It is important to notice ops.